src/components/Note/EditableNoteMentions.tsx

```diff
@@ -0,0 +1,187 @@
+  onSave: (note: Note) => void;
+  rows?: number;
+  saving?: boolean;
+  saveLabel?: React.ReactNode;
+  savingLabel?: React.ReactNode;
+};
+
+export const EditableNoteMentionsDefaultProps = {
+  className: 'bg-white mb-3',
+  dateFormat: 'ddd, MMMM D, YYYY "at" h:mm A',
+  mentionableUsers: [],
+  rows: 4,
+  saving: false,
+  saveLabel: 'Save',
+  savingLabel: 'Saving...',
+  showTimezone: true,
+};
+
+const EditableNoteMentions: FC<EditableNoteMentionsProps> = ({
+  className = EditableNoteMentionsDefaultProps.className,
+  dateFormat = EditableNoteMentionsDefaultProps.dateFormat,
+  mentionableUsers = EditableNoteMentionsDefaultProps.mentionableUsers,
+  rows = EditableNoteMentionsDefaultProps.rows,
+  saving = EditableNoteMentionsDefaultProps.saving,
+  saveLabel = EditableNoteMentionsDefaultProps.saveLabel,
+  savingLabel = EditableNoteMentionsDefaultProps.savingLabel,
+  showTimezone = EditableNoteMentionsDefaultProps.showTimezone,
+  ...props
+}) => {
+  const { children, note, onCancel, onChange, onSave } = props;
+  const { errors, text } = note;
+
+  const [noteBodyId] = useState(uniqueId('note_body_'));
+
+  useEffect(() => {
+    injectTribute(mentionableUsers, noteBodyId);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  return (
+    <Card className={className}>
+      <NoteHeader note={note} dateFormat={dateFormat} showTimezone={showTimezone} />
+      <CardBody>
+        <FormLabelGroup feedback={errors} stacked>
+          <Input
+            id={noteBodyId}
+            autoFocus
+            className="js-editable-note_text"
+            disabled={saving}
+            rows={rows}
+            invalid={!!errors}
+            type="textarea"
+            value={text}
+            onChange={(event) => onChange(event, note)}
+          />
+        </FormLabelGroup>
+        {children}
+        <ButtonToolbar className="mt-3 mb-0">
+          <Button
+            className="js-editable-note_save"
+            color="primary"
+            disabled={saving}
+            onClick={() => onSave(note)}
+          >
+            {saving ? savingLabel : saveLabel}
+          </Button>
+          <Button
+            className="js-editable-note_cancel"
+            disabled={saving}
+            onClick={() => onCancel(note)}
+          >
+            Cancel
+          </Button>
+        </ButtonToolbar>
+      </CardBody>
+    </Card>
+  );
+};
+
+EditableNoteMentions.defaultProps = {
+  className: EditableNoteMentionsDefaultProps.className,
+  dateFormat: EditableNoteMentionsDefaultProps.dateFormat,
+  mentionableUsers: EditableNoteMentionsDefaultProps.mentionableUsers,
+  rows: EditableNoteMentionsDefaultProps.rows,
+  saving: EditableNoteMentionsDefaultProps.saving,
+  saveLabel: EditableNoteMentionsDefaultProps.saveLabel,
+  savingLabel: EditableNoteMentionsDefaultProps.savingLabel,
+  showTimezone: EditableNoteMentionsDefaultProps.showTimezone,
+};
```

Comment:
Does this work for you?
```suggestion
EditableNoteMentions.defaultProps = EditableNoteMentionsDefaultProps;
```